datahub-web-react/src/app/ingest/source/conf/kafka/kafka.ts

```diff
@@ -23,8 +23,10 @@ source:
             #     basic.auth.user.info: "\${REGISTRY_API_KEY_ID}:\${REGISTRY_API_KEY_SECRET}"
```

Comment:
can you replace the `placeholderRecipe` above with this below:
```
source:
    type: kafka
    config:
        connection:
            consumer_config:
                security.protocol: "SASL_SSL"
                sasl.mechanism: "PLAIN"
        stateful_ingestion:
            enabled: true
```

This is how we're going to set default values on these forms

Reply:
Done!!